VirtoCommerce.CatalogModule.Web/Scripts/blades/bulk/bulk-action-registrar.js

```diff
@@ -1,13 +0,0 @@
-angular.module('virtoCommerce.catalogModule')
```

Comment:
We should not remove it. It is for linking server actions with the customizable UI.

Reply:
Also please replace element if the one with the same actionName already exists.